src/lib/backpack/sprite-payload.js

```diff
@@ -1,23 +1,29 @@
 import jpegThumbnail from './jpeg-thumbnail';
 
-const spritePayload = (sprite, vm) => vm.exportSprite(
-    sprite.id,
-    'base64'
-).then(zippedSprite => {
-    const payload = {
-        type: 'sprite',
-        name: sprite.name,
-        mime: 'application/zip',
-        body: zippedSprite,
-        // Filled in below
-        thumbnail: ''
-    };
+const spritePayload = (id, vm) => {
```

Comment:
I don't understand this change, is it an additional performance bump, or related to the throttling?

Reply:
Good question. This was a necessary bugfix that we needed to make before the throttledpropertyhoc could work. There was an overly greedy change to the dragPayload which was causing shouldComponentUpdate to always return true.